zigbee2mqtt-edge/config.json

```diff
@@ -55,6 +55,7 @@
     "passlist": [],
     "queue": {},
     "frontend": {
+      "host": "0.0.0.0",
```

Comment:
Have you tested with this as string? I was not able to test it yet, just made the fix based on the docs.

Reply:
@ciotlosm I have tested it again and the problem is not the lack of a default value but a wrong type in the schema. :/

Reply:
Seems I was also tired enough that I didn't notice. Thanks for re-checking